ivy/functional/backends/torch/experimental/statistical.py

```diff
@@ -47,3 +47,35 @@ def nanmean(
 
 
 nanmean_support_native_out = True
+
+
+def nanquantile(
+    a: torch.Tensor,
+    q: Union[int, float],
+    /,
+    *,
+    axis: Optional[Union[int, Tuple[int]]] = None,
+    keepdims: Optional[bool] = False,
+    interpolation: Optional[str] = None,
+    out: Optional[torch.tensor] = None,
+) -> torch.Tensor:
+    if hasattr(axis, "__iter__"):
```

Comment:
This shouldn't be used, the whole implementation should simply just be `return torch.nanquantile(
            a,
            q=q,
            dim=axis,
            keepdim=keepdims,
            interpolation=interpolation,
            out=out,
        )`